shared/src/main/java/uk/gov/di/authentication/shared/entity/ErrorResponse.java

```diff
@@ -67,7 +67,8 @@ public enum ErrorResponse {
     ERROR_1053(1053, "Account Interventions API Bad Gateway"),
     ERROR_1054(1054, "Account Interventions API Gateway Timeout"),
     ERROR_1055(1055, "Account Interventions API Unexpected Error"),
-    ERROR_1056(1056, "User not found or no match");
+    ERROR_1056(1056, "User not found or no match"),
+    ERROR_1057(1057, "User not found or no match");
```

Comment:
Are these different?

Reply:
They are meant to be, didn't change the message. Will update now